lib/src/main/kotlin/starknet/data/types/Felt.kt

```diff
@@ -16,6 +18,10 @@ data class Felt(val value: BigInteger) {
         }
     }
 
+    override fun toString(): String {
```

Comment:
We probably would like to choose between hex and dec string

Reply:
This is the default toString method, we can custom ones later